### ISSUE TITLE
Request: Add an interactive profiling view (like quak) to explore tabular data 

### ISSUE THREAD

**Issue by rluthi** [koaning/bulk#74]
To explore tabular data, it would be nice to have in viewing panel on the right, a minimalist profiling summary of the selected points vs. the whole population (all the points).

I quite like how @manzt does it in the [quak](https://github.com/manzt/quak) interactive data frame viewer (selected points in blue and whole population in grey):

![image](https://github.com/user-attachments/assets/900742d2-0ee4-436e-b185-ddd03e629211)

**Comment by koaning** [comment]
What might such a view offer that quak currently does not?

**Comment by rluthi** [comment]
My use-case is exploring a 2-D embedding of my data to identify interesting clusters and describe with a subset of "profiling variables". `quak` is great when you already have your segments defined. But, in the exploration phase, it requires re-executing the cell with the widget each time a new selection is made. 

With `quak`, there is also a lot of horizontal scrolling to do on larger datasets. I like the idea of having the profiling view in the right pane of the `bulk` widget with all of my "profiling variables" visible on screen at the same time.

**Comment by koaning** [comment]
Ah, so in your use-case quak would more be a data viewer, not a selector for other views so to say? In bulk land the UMAP chart is a selector element and the text on the side is a viewer.

**Comment by rluthi** [comment]
Exactly! I would like to be able to select points in a 2D UMAP scatter plot selector (like in your text dataset demo) and have it synced with a `quak`-like viewer panel with the profiling panel

**Comment by rluthi** [comment]
Very roughly:
![image](https://github.com/user-attachments/assets/4a0ed396-5059-41d4-ba0b-0bd041bcf0e3)

**Comment by koaning** [comment]
Would you still not want a table preview at the bottom there? This feels relatively easy to implement. Might have a go at it later today.

**Comment by rluthi** [comment]
Table preview would be nice if there is enough space. Could be an option...

**Comment by koaning** [comment]
A lot of it kind of depends on the number of columns. Once you start having a lot of them it because hard to see what the selection is really about. 

Just a rough feeling, in your day to day, how many columns do you have? Also, what types? A lot of categoricals?

**Comment by rluthi** [comment]
8 to 15 columns, mostly categories /boolean (~70% of columns) the rest are numerical
